src/ResourceManagement/Azure.Fluent/Microsoft.Azure.Management.Fluent/project.json

```diff
@@ -26,7 +26,8 @@
     "Microsoft.Azure.Management.KeyVault.Fluent": "[1.0.0-beta3, 2.0)",
     "Microsoft.Azure.Management.Network.Fluent": "[1.0.0-beta3, 2.0)",
     "Microsoft.Azure.Management.ResourceManager.Fluent": "[1.0.0-beta3, 2.0)",
-    "Microsoft.Azure.Management.Storage.Fluent": "[1.0.0-beta3, 2.0)"
+    "Microsoft.Azure.Management.Storage.Fluent": "[1.0.0-beta3, 2.0)",
+    "Microsoft.Azure.Management.TrafficManager.Fluent": "1.0.0-beta3"
```

Comment:
>Fluent": "1.0.0-beta3" [](start = 47, length = 22)

please fix the version reference by following the same pattern as the libraries above.